common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -427,6 +427,13 @@ nav.sequence-bottom {
       }
     }
   }
+
+#seq_content {
+
+  &:focus,
+  &:active {
+    outline: none;
```

Comment:
This doesn't seem to be working? I am getting a visible focus ring in FF, Chrome and Safari on Mac